src/set_relation/TermPartOrdGraph.cc

```diff
@@ -90,6 +91,17 @@ int TermPartOrdGraph::findOrInsertTermId(const Term* t) {
         } else {
             termId = mVarTerm2IntMap[*vterm];
         }
+    } else if (temp->type()=="Term") {
```

Comment:
Put comments here about why putting negative of the constant in the table as well.